docs/ethsimulatev1-notes.md

```diff
@@ -0,0 +1,229 @@
+# Multicall
+This document contains some extra information that couldn't be fit to the specification document directly.
+
+## Default block values
+Unlike `eth_call`, `eth_simulateV1`'s calls are conducted inside blocks. We don't require user to define all the fields of the blocks so here are the defaults that are assumed for blocks parameters:
+
+| parameter name | default value |
+-----------------|-----------------------
+| prevRandao | `0x0000000000000000000000000000000000000000000000000000000000000000` |
+| feeRecipient | `0x0000000000000000000000000000000000000000` |
+| mixHash | `0x0000000000000000000000000000000000000000000000000000000000000000` |
+| nonce | `0x0` |
+| extraData | `0x0000000000000000000000000000000000000000000000000000000000000000` |
+| difficulty | The same as the base block defined as the second parameter in the call |
+| gasLimit | The same as the base block defined as the second parameter in the call |
+| hash | Calculated normally, except for phantom blocks, see below Phantom block section |
+| parentHash | Previous blocks hash (the real hash, or phantom blocks hash) |
+| timestamp | The timestamp of previous block + 1 |
+| baseFeePerGas | Calculated on what it should be according to Ethereum's spec. Note: baseFeePerGas is not adjusted in the phantom blocks. |
+| sha3Uncles | Empty trie root |
+| withdrawals | Empty array |
+| uncles | Empty array |
+| blobBaseFee | Calculated on what it should be according to EIP-4844 spec. Note: blobBaseFee is not adjusted in the phantom blocks. |
+| number | Previous block number + 1 |
+| logsBloom | Calculated normally. ETH logs are not part of the calculation |
+| receiptsRoot | Calculated normally |
+| transactionsRoot | Calculated normally |
+| size | Calculated normally |
```

Comment:
`size` is not part of the block header (so I also don't see why it is necessary here?)